### PR TITLE
KO: Translation for ko/beta/guide/autograph.ipynb

### PR DESCRIPTION
**Pull request by BY1994**
I translated site/en/r2/guide/autograph.ipynb

If there are any problems with this translation, please feel free to let me know.

Original version: https://github.com/tensorflow/docs/blob/master/site/en/r2/guide/autograph.ipynb

### CODE REVIEW

File: site/ko/beta/guide/autograph.ipynb [hunk 1]
```diff
@@ -0,0 +1,766 @@
...
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/beta/guide/autograph\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />TensorFlow.org 에서 보기</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/r2/guide/autograph.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />구글 코랩(Google Colab)에서 실행하기</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/r2/guide/autograph.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />깃헙(GitHub) 소스 보기</a>\n",
```
Comment (cre8tor) [inline site/ko/beta/guide/autograph.ipynb]: the link needs correction also. :)

File: site/ko/beta/guide/autograph.ipynb [hunk 2]
```diff
@@ -0,0 +1,766 @@
...
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/en/r2/guide/autograph.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />notebook 다운로드</a>\n",
```
Comment (cre8tor) [inline site/ko/beta/guide/autograph.ipynb]: As far as I know, Korean translation of notebook does not support "notebook download".
Maybe it should be deleted?

File: site/ko/beta/guide/autograph.ipynb [hunk 3]
```diff
@@ -0,0 +1,766 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "CydFK2CL7ZHA"
+      },
+      "source": [
+        "TF 2.0 버전은 즉시 실행 (eager execution)의 편리함과 TF 1.0의 파워를 합쳤습니다. 이러한 결합의 중심에는 `tf.function` 이 있는데, 이는 파이썬 문법의 일부를 이식 가능하고 높은 성능의 텐서플로 그래프 코드로 변환시켜줍니다. \n",
```
Comment (cre8tor) [inline site/ko/beta/guide/autograph.ipynb]: I suggest '파워' -> '성능'

File: site/ko/beta/guide/autograph.ipynb [hunk 4]
```diff
@@ -0,0 +1,766 @@
...
+        "\n",
+        "`tf.function`의 멋지고 새로운 특징은 오토그래프 (AutoGraph)입니다. 이는 자연스러운 파이썬 문법을 활용해서 그래프 코드를 작성할 수 있도록 돕습니다. 오토그래프로 사용할 수 있는 파이썬 특질들의 목록을 보려면 [오토그래프 지원 범위](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/autograph/LIMITATIONS.md)를 참고하세요. `tf.function`에 관한 더 자세한 내용을 확인하려면 RFC [TF 2.0: Functions, not Sessions](https://github.com/tensorflow/community/blob/master/rfcs/20180918-functions-not-sessions-20.md)을 참고하세요. 오토그래프에 대한 더 자세한 내용은 `tf.autograph`를 참고하세요.\n",
```
Comment (cre8tor) [inline site/ko/beta/guide/autograph.ipynb]: Here's a typo. '특질' -> '특징'

File: site/ko/beta/guide/autograph.ipynb [hunk 5]
```diff
@@ -0,0 +1,766 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "DqeefLGNXjZQ"
+      },
+      "source": [
+        "만일 여러분의 코드가 여러 함수들을 포함하고 있다면, 그것들에 모두 데코레이터를 붙일 필요는 없습니다. 데코레이터가 붙은 함수로부터 불린 모든 함수들은 그래프 모드에서 동작합니다."
```
Comment (cre8tor) [inline site/ko/beta/guide/autograph.ipynb]: I think '호출된' is more widely used for 'called'
'불린' -> '호출된'

File: site/ko/beta/guide/autograph.ipynb [hunk 6]
```diff
@@ -0,0 +1,766 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "yQvg6ZSKWyqE"
+      },
+      "source": [
+        "작은 ops들을 많이 포함한 그래프의 경우 함수들은 즉시 실행 코드 (eager code) 보다 더 빠르게 동작합니다. 하지만 큰 ops들을 조금 포함한 그래프의 경우 (컨볼루션 등), 그렇게 빠른 속도 향상은 기대하기 어렵습니다.\n"
```
Comment (cre8tor) [inline site/ko/beta/guide/autograph.ipynb]: 'ops' here means operations. so it needs to be translated '연산'
'작은 ops' -> '작은 연산들을', '큰 ops' -> '무거운 연산'

File: site/ko/beta/guide/autograph.ipynb [hunk 7]
```diff
@@ -0,0 +1,766 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "ohbSnA79mcJV"
+      },
+      "source": [
+        "## 파이썬의 제어 흐름 사용하기\n",
+        "\n",
+        "`tf.function` 내에서 데이터 기반 제어 흐름을 사용할 때, 파이썬의 제어 흐름 문을 사용할 수 있고, 오토그래프 기능은 그것들을 모두 적절한 텐서플로 ops로 변환할 수 있습니다. 예를 들어, `if` 문은 `Tensor`를 기반으로 작동해야할 때 `tf.cond()` 로 변환될 수 있습니다. \n",
```
Comment (cre8tor) [inline site/ko/beta/guide/autograph.ipynb]: 'ops' here also needs translation :)

File: site/ko/beta/guide/autograph.ipynb [hunk 8]
```diff
@@ -0,0 +1,766 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "AtDaLrbySw4j"
+      },
+      "source": [
+        "또한 오토그래프는 어드밴스드 유저를 위해 낮은 수준의 API를 제공합니다. 예를 들어, 여러분은 생성된 코드를 확인하기 위해 다음과 같이 작성할 수 있습니다. "
```
Comment (cre8tor) [inline site/ko/beta/guide/autograph.ipynb]: according to existing translation, 
'어드밴스드 유저' -> '고급 사용자'

File: site/ko/beta/guide/autograph.ipynb (hunk 2 above)
Comment (BY1994) [reply site/ko/beta/guide/autograph.ipynb]: I noticed that the function (notebook download) is available in other languages.
Is there any way to check whether the download of the notebook is possible or not?

Comment (BY1994) [reply site/ko/beta/guide/autograph.ipynb]: @cre8tor 
I noticed that the function (notebook download) is available in other languages.
Is there any way to check whether the download of the notebook is possible or not?

Comment (cre8tor) [reply site/ko/beta/guide/autograph.ipynb]: @BY1994 lamberta said 'download notebook' button will be added all at once when they are available. ( #671 ) You don't have to add the button at the moment.

Comment (BY1994) [reply site/ko/beta/guide/autograph.ipynb]: @cre8tor Thanks! I removed the button.

**Pull request merged**